compiler/src/yul/runtime/functions.rs

```diff
@@ -0,0 +1,161 @@
+use yultsur::*;
+
+/// Returns all functions that should be available during runtime.
+pub fn all() -> Vec<yul::Statement> {
+    vec![
+        avail(),
+        alloc(),
+        alloc_mstoren(),
+        free(),
+        ccopy(),
+        mcopy(),
+        scopy(),
+        mloadn(),
+        sloadn(),
+        cloadn(),
+        mstoren(),
+        sstoren(),
+        dualkeccak256(),
```

Comment:
This should be removed in favor of a generalized `keccak256` function available on all fixed size types.

``` rust
let a = literal_expression! { 42 };
let b = literal_expression! { 12 };
let my_tuple = FixedSize::Tuple(vec![FixedSize::U256, FixedSize::U256]);

my_tuple.keccak256(a, b) // equivalent to dualkeccak256()
```